tests/Component/ComponentLoaderTest.php

```diff
@@ -13,7 +13,7 @@ class ComponentLoaderTest extends \PHPUnit_Framework_TestCase
 {
     private function loadAtoms ()
     {
-        $loader = new ComponentLoader(__DIR__ . "/../fixtures/Resources/views");
+        $loader = new ComponentLoader(__DIR__ . "/../app/src/LayoutBundle/Resources/views");
```

Comment:
Just thinking out loud: I'm not entirely sure I like the hardcoded path here. I'd rather have it resolved by Symfony via `@Layout/...` etc. However, the drawback here would be to take a dependency on `KernelInterface`'s `locateResource` method.
Maybe it's not even possible to use that method since you're pointing to a directory instead of a resource file.